packages/nuxt/src/runtime/utils.ts

```diff
@@ -26,3 +30,26 @@ export function extractErrorContext(errorContext: CapturedErrorContext): Context
 
   return dropUndefinedKeys(structuredContext);
 }
+
+/**
+ * Adds Sentry tracing <meta> tags to the returned html page.
+ *
+ * Exported only for testing
+ */
+export function addSentryTracingMetaTags(head: NuxtRenderHTMLContext['head']): void {
+  const activeSpan = getActiveSpan();
+  const rootSpan = activeSpan ? getRootSpan(activeSpan) : undefined;
+
+  if (rootSpan) {
```

Comment:
cc @Lms24 I remember you wrote this somewhere at some point - should we also inject this for Tracing Without Performance? We do not do this anywhere today I believe, but there was talk about _starting_ to do this?

Reply:
hmm that's a great point! You're right, we should also inject meta tags for TwP mode. @s1gr1d I propose we merge this as-is and follow up on generally changing the logic here across SDKs. I'll search if we already have an issue for this and otherwise create one to track.

Reply:
I looked it up: This issue already exists (https://github.com/getsentry/sentry-javascript/issues/8520). I added Nuxt as an item to track but there was a concern raised that I think held us off from implementing